README.md

```diff
@@ -6,8 +6,7 @@
 Du bist Manager eines Fernsehsenders in diesem __kostenlosen Spiel__ . Lenke was gesendet wird und wann. Verdiene Geld durch Werbeeinnahmen und treibe die Konkurrenz in den Bankrott. Hole Zuschauer vor die Flimmerkiste in dem Du brandheiße Nachrichtenstories sendest und mit spannenden Livesendungen Millionen von Mitmenschen am Einschlafen hinderst. Fertigprodukte aus Hollywood sind nicht so dein Ding? Tja, Eigenproduktionen könnten Quotenhits werden - oder aber teure Millionengräber.
 
 ### English
-In this __free game__ you are the manager of a television channel. Control what gets broadcasted and when. Earn Money with commercial breakups and drive your opponents into bankruptcy. Broadcast top stories in your news block or live shows keeping your audience from falling asleep. You dislike convenience products made in Hollywood - no problem - running your own products might lead to new stars of the ratings - or a costly white elephant.
-
+TVTower is a __free game__ where you are the manager of a television channel. Strategically schedule broadcast messages. Earn money with commercial breaks and drive your competition to bankruptcy. Broadcast breaking news stories or live shows to keep viewers from sleeping. Not interested in commercial products from hollywood? No problem, running your own products may lead to increased viewership ratings, or a luxury white elephant.
```

Comment:
The "new stars of the ratings" tried to express that our new custom production might become some kind of "rating blockbuster" - a production with enormous ratings.
Your translation "increased viewership ratings" just expresses... an increased rating.

costly/luxury white elephant: good catch